src/handlers/slack/handle-command.ts

```diff
@@ -25,11 +26,18 @@ const handleResponse = async (
   logger: Logger
 ): Promise<void> => {
   try {
-    const { message, destination } = await response;
-    const respondArguments = {
+    const { message, destination, metaData } = await response;
+    const respondArguments: RespondArguments = {
       text: message,
       response_type: getResponseType(destination),
     };
+    if (metaData && metaData.Message) {
+      respondArguments.attachments = [
```

Comment:
I don't think `attachments` is the right thing here.

I think we could use `Block quote` instead. Although I'm not sure how we'd also make the description look nice when using `/locks`, perhaps a newline and italics? Would have to have a think/experiment.

In slack `attachments` looks very similar to a `Block quote`
![image](https://user-images.githubusercontent.com/10026538/153778451-ba18084a-7b45-442c-a63f-c99621c5f56f.png)
which is what we use with `/locks`
![image](https://user-images.githubusercontent.com/10026538/153778489-c56f0e1d-0030-4a04-a52c-8459fb9f9c50.png)

Except you can remove the attachment, which is probably undesirable
![image](https://user-images.githubusercontent.com/10026538/153778584-f32a485f-d4c9-4bc8-b0a3-4af26c87313e.png)
![image](https://user-images.githubusercontent.com/10026538/153778602-754717b1-766f-4242-9ae4-0ae4189c0174.png)